guides/assets/esa-initial-flow.txt

```diff
@@ -0,0 +1,40 @@
+sequenceDiagram
+    participant User
+    participant App
+    participant Application Route
+    participant Authenticated Route
+    participant Unauthenticated Route
+    participant Session Service
+    participant Torii Provider
+    participant GitHub
+    participant Token Exchanger
+    User->>App: Use https://git-tool.com
+    App->>Application Route: transitionTo(application)
```

Comment:
Agreed. This assumes the Application Route uses the `ApplicationRouteMixin` and is a deliberate simplification. Is there a better way to indicate the use of the mixin?

Do you think
```
App->>Application Route: beforeModel
Application Route->>Application Route: transitionTo(application)
```
would be more accurate?